resources/dicts/patrols/new_cat.json

```diff
@@ -32,87 +55,178 @@
     ],
-    "min_cats": 1,
+    "min_cats": 2,
     "max_cats": 6,
     "win_skills": ["good speaker", "great speaker", "excellent speaker"],
     "antagonize_text": "Your patrol drives the kittypet off of the territory.",
     "antagonize_fail_text": "The kittypet is taken aback by their hostility and decides that Clan life is not for them."
 
+},
+{
+    "patrol_id": "gen_bord_newcatkittypetsolo2",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["border", "new_cat_warrior_kittypet_litter1", "reputation", "meeting"],
+    "intro_text": "p_l finds a kittypet who is interested in joining the Clan.",
+    "decline_text": "p_l decides not to confront the kittypet.",
+    "chance_of_success": 40,
+    "exp": 10,
+    "success_text": [
+        "p_l convinces the kittypet to join the Clan.",
+        "p_l convinces the kittypet to join the Clan, and they join with their litter of kits."
+    ],
+    "fail_text": [
+        "The description of Clan life frightens the kittypet."
+    ],
+    "min_cats": 1,
+    "max_cats": 1,
+    "win_skills": ["good speaker", "great speaker", "excellent speaker"],
+    "antagonize_text": "p_l drives the kittypet off of the territory.",
+    "antagonize_fail_text": "The kittypet is taken aback by p_l's hostility and decides that Clan life is not for them."
+
 },
 {
     "patrol_id": "gen_bord_newcatthunderpath3",
     "biome": "Any",
     "season": "Any",
     "tags": ["border", "new_cat_warrior", "new_cat_injury", "nc_blunt_force_injury", "reputation", "no_change_fail_rep"],
     "intro_text": "r_c finds a wounded cat near the Thunderpath.",
+    "decline_text": "They leave the wounded cat alone.",
     "success_text": [
         "The patrol approaches the injured cat and sees signs of life. They gingerly pick up the cat and take them back to camp to be treated by the medicine cat. The cat is grateful for their help and decides to join the Clan."
     ],
     "fail_text": [
-        "As r_c inspects the cat, they find that they are already hunting with their ancestors.",
+        "As r_c inspects the cat, {PRONOUN/r_c/subject} {VERB/r_c/find/finds} that they are already hunting with their ancestors.",
         "The cats attempt to carry the cat back to camp, but the cat passes on the way. They stop to bury the cat before continuing home."
     ],
-    "decline_text": "They leave the wounded cat alone.",
     "antagonize_text": "Your patrol drives the cat off of the territory. It's not c_n's problem.",
     "antagonize_fail_text": "The patrol tries to drive the cat away, but they trip and stumble over a loose twig. They lay on the ground and don't get back up. The patrol takes a bit of time to bury them - just to keep the scavengers away.",
     "chance_of_success": 40,
     "exp": 10,
     "win_skills": ["smart", "very smart", "extremely smart"],
-    "min_cats": 1,
+    "min_cats": 2,
     "max_cats": 6
 
+},
+{
+    "patrol_id": "gen_bord_newcatthunderpathsolo3",
```

Comment:
in order to make sure this works for the patrol sprite, you should remove 'solo' from the ID and just increment the number